frontend/src/components/molecules/ModalWindow/ModalWindow.tsx

```diff
@@ -0,0 +1,53 @@
+import { useRef } from 'react';
+import { useOutsideClick } from 'hooks';
+import { colors } from 'styles/colors';
+import { BUTTON_TITLE } from 'Types/enums';
+import { IModalWindow } from './ModalWindow.model';
+import {
+  StyledButton,
+  StyledCancelButton,
+  StyledContainer,
+  StyledParagraph,
+  StyledTitle,
+  StyledWindow,
+  StyledWrapper,
+} from './ModalWindow.styled';
+
+const ModalWindow = ({
+  title,
+  isActive,
+  description,
+  handleCancel,
+  handleConfirm,
+}: IModalWindow): JSX.Element => {
+  const modalRef = useRef(null);
+  useOutsideClick(modalRef, () => isActive && handleCancel());
+
+  const handleButtonColor = (buttonTitle: string) => {
+    if (buttonTitle === BUTTON_TITLE.DELETE) {
```

Comment:
I think it should be realize in styled component, not in the function inside component